geograypher/meshes/meshes.py

```diff
@@ -1344,7 +1340,13 @@ def get_rasterization_results_pytorch3d(
         ).to(self.device)
 
         # Ensure that a pytorch3d mesh exists
-        self.create_pytorch3d_mesh(batch_size=len(p3d_cameras))
+        self.create_pytorch3d_mesh()
+
+        # Number of cameras should be equal to number of meshses
+        batch_size = len(p3d_cameras)
+        if batch_size != len(self.pytorch3d_mesh):
+            self.pytorch3d_mesh = self.pytorch3d_mesh.extend(batch_size)
```

Comment:
I don't think you want to set `self.pytorch3d_mesh` to the extended version. In this case, it will keep getting longer on each iteration.

Reply:
Sorry, this was supposed to be in the first review.